fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -163,7 +163,12 @@ def self.xcode_version
 
     # @return true if Xcode version is higher than 8.3
     def self.xcode_at_least?(version)
-      FastlaneCore::UI.user_error!("Unable to locate Xcode. Please make sure to have Xcode installed on your machine") if xcode_version.nil?
+      if self.ci?
```

Comment:
Do we actually want to make this `self.mac?`? 🤔  Since this would be nice to see on any linux/windows run and not necessarily just on CI?

Reply:
Not really, this is explicit communication to a developer. As I mentioned above, this is only 1 location out of hundreds that use Xcode (although one of the most "popular" ones), so this is also not complete.

Reply:
Why not? Maybe a Linux user clones the branch of someone using macOS that added some tests, but forgot to specify that flag. In which case, for that other person, the tests pas, but for the Linux user, they fail. And with this PR and Josh's change, he will knew exactly what the problem is, when running the tests locally, because the error message is clearer.

And back to your point, if xcodebuild is used from multiple places (which I'm not doubting), how does it work on Linux? And, we could fix it (e.g. make them all pass through a single function) in another PR, right? Meaning, if this is ok, it could be merged, right?